src/main/java/org/broadinstitute/dsm/route/PatchRoute.java

```diff
@@ -263,6 +272,27 @@ else if (participantDataId != null) {
                         }
                         return new Result(200, new GsonBuilder().serializeNulls().create().toJson(map));
                     }
+                    else if (Patch.DDP_PARTICIPANT_ID.equals(patch.getParent())) {
+                        //new additional value for pt which is not in ddp_participant and ddp_participant_record table yet
+                        DDPInstance ddpInstance = DDPInstance.getDDPInstance(patch.getRealm());
+                        String participantId = inTransaction(conn -> {
+                            String newParticipantId = MedicalRecordUtil.writeParticipantIntoDB(conn, patch.getParentId(), ddpInstance.getDdpInstanceId(), 0, "", patch.getUser());
+                            MedicalRecordUtil.writeNewRecordIntoDb(conn, DDPMedicalRecordDataRequest.SQL_INSERT_PARTICIPANT_RECORD, patch.getParentId(), ddpInstance.getDdpInstanceId());
+                            return newParticipantId;
+                        });
+                        if (StringUtils.isNotBlank(participantId)) {
+                            DBElement dbElement = patchUtil.getColumnNameMap().get(patch.getNameValue().getName());
+                            if (dbElement != null) {
+                                Patch.patch(participantId, patch.getUser(), patch.getNameValue(), dbElement);
+                            }
+                            else {
+                                throw new RuntimeException("DBElement not found in ColumnNameMap: " + patch.getNameValue().getName());
+                            }
+                            Map<String, String> map = new HashMap<>();
+                            map.put(PARTICIPANT_ID, participantId);
+                            return new Result(200, new GsonBuilder().serializeNulls().create().toJson(map));
```

Comment:
To be honest, I really don't understand using `Result` instead of directly return result and let `SparkJava` to automatically transform it into json and send it back, if main idea is to set response code we could done it using `response.status()` method for setting specific codes.

Reply:
inherited logic from lddp, I agree it is time to change that!